pkg/internal/checkup/executor/testpmd/console_test.go

```diff
@@ -0,0 +1,134 @@
+package testpmd_test
```

Comment:
Could you please add the license comment?

Reply:
DONE